src/Components/IndividualReview.jsx

```diff
@@ -10,26 +10,65 @@ const IndividualReview = () => {
   const { ID } = useParams();
   const [review, setReview] = useState({});
   const [comments, setComments] = useState([]);
-
-  console.log(ID, "<<ID");
+  const [votes, setVotes] = useState(0);
+  const [err, setErr] = useState(null);
 
   useEffect(() => {
     axios
       .get(`https://josies-games.herokuapp.com/api/reviews/${ID}`)
       .then((res) => {
         setReview(res.data.review);
+        setVotes(res.data.review.votes);
```

Comment:
it's not a good idea to start storing the same information in two places. Which one is the 'real' one?

`votes` is both in `review` *and* `votes` - this could become really buggy (and confusing for fellow devs!)